### ISSUE TITLE
Training set

### ISSUE THREAD

**Issue by longkhanh-fam** [ms-dot-k/Lip-to-Speech-Synthesis-in-the-Wild#13]
Hi,
Which subsets do you use to train in lrs2 and lrs3, only train-val subset or both train-val and pretrain? And what is the used ratio between subsets.
Do you combine the train and pre-train subset or train them seperately, if seperately, what is the order of training subsets?

Thank you for your answer

**Comment by ms-dot-k** [comment]
Hello,
Both pretrain and train splits are employed for training by merging the file lists.
For LRS2, we employed the official splits. 
For LRS3, please refer to ['./data/LRS3'](https://github.com/ms-dot-k/Lip-to-Speech-Synthesis-in-the-Wild/tree/main/data/LRS3).

**Comment by longkhanh-fam** [comment]
@ms-dot-k , thank you for your response
For the LRS3 dataset, is the validation set split according to the original partition, or did you implement a custom split for it?

**Comment by ms-dot-k** [comment]
As described in README and paper, we followed the splits of SVTS (https://arxiv.org/abs/2205.02058).

**Comment by longkhanh-fam** [comment]
Since SVTS hasn't made their source code publicly available, I just wonder about how you obtained the validation file.  Just for a fair comparison as I noticed that your validation.txt file includes pretrain samples.

Also, I genuinely appreciate your support —thank you!

**Comment by ms-dot-k** [comment]
Sure, we directly received the lists from the author of SVTS.
